spec/lite_spec_helper.rb

```diff
@@ -67,6 +68,7 @@
 require 'support/sdam_monitoring'
 require 'support/crud'
 require 'support/command_monitoring'
+require 'support/connection_monitoring_and_pooling'
```

Comment:
Please rename this file to `support/cmap` for consistency.

Reply:
`support/server_discovery_and_monitoring.rb` should be renamed to `support/sdam.rb` for the same reason.

Reply:
Done